dvc/repo/__init__.py

```diff
@@ -448,6 +449,19 @@ def used_objs(
     def stages(self):  # obsolete, only for backward-compatibility
         return self.index.stages
 
+    @property
+    def ignore_revs_file(self):
+        if self.dvc_dir:
+            return self.fs.path.join(self.dvc_dir, "ignore_revs")
+
+    @property
+    def ignore_revs(self):
```

Comment:
We should clarify what is `ignore_revs` file for an external repository. Do we need to read from the default branch or just the one we checked out? If we need to read from default branch, it might not be possible with shallow clones AFAICT.


Also what should happen if user asks for that particular revision (eg: `plots diff <ignored_rev>`)?

Reply:
> We should clarify what is ignore_revs file for an external repository.

Not sure I follow. Could you elaborate the use case?

> Do we need to read from the default branch or just the one we checked out? If we need to read from default branch, it might not be possible with shallow clones AFAICT.

I believe we should read from the current revision. Treat `ignore-revs` as if it was part of `config`.

> Also what should happen if user asks for that particular revision (eg: plots diff <ignored_rev>)?

We will skip the revision.
The use case is to use `ignore_revs` for broken revisions. 
If a revision is broken, `plots diff` won't work (see https://github.com/iterative/dvc/issues/6150) so if we don't skip user will encounter the error?